src/cmdc_tools/datasets/official/VA/data.py

```diff
@@ -0,0 +1,82 @@
+import pandas as pd
+
+from ...base import DatasetBaseNoDate
+from ..base import ArcGIS
+
+
+class Virginia(ArcGIS, DatasetBaseNoDate):
+    ARCGIS_ID = "BkFxaEFNwHqX3tAw"
+
+    def get(self):
+        df = self.get_all_sheet_to_df(service="VT_Counties_Cases", sheet=0, srvid=1)
+
+        renamed = df.rename(columns={
+            "CntyLabel": "county",
+            "CNTYGEOID": "fips",
+            "Cases": "cases_total",
+            "Deaths": "deaths_total",
+            "DateUpd": "dt"
+        })
+        # Convert Timestamps
+        renamed["dt"] = pd.to_datetime(renamed["dt"].map(
+            lambda x: pd.datetime.fromtimestamp(x/1000).date()
+        ))
+
+        county_summ = self._get_county_summary()
+
+
+        renamed = renamed[[
+            "county",
```

Comment:
Is there a reason to keep the `county` label around?

Reply:
Wasn't sure if you wanted it or not. I left it. I'll let @cc7768 and @sglyon decide if they want to keep it or not